Toolset/palettes/standalone settings/revstandalonesettingsiosbehavior.livecodescript

```diff
@@ -419,13 +434,29 @@ end chooseImageFileWithDimension
 
 on tipDisplayUpdate
    if the visible of group "iOSCommercialTip" then
-      set the topLeft of group "iosBuild" to 12,136
-      set the topLeft of group "iOS Settings" to -1,136
-      set the cSize of this card to 1024,726
+      set the topLeft of group "buildbutton" to 50,127
+      set the topLeft of group "iosSettings" to 5,127
+      repeat for each line tLine in the text of button "iOSTabMenu"
+         local tName
+         put tLine into tName
+         if tName is "Requirements" then
+            put "Requirement" into tName
+         end if
+         set the topLeft of group tName to 12,178
+      end repeat 
+      set the cSize of this card to 754,670
    else 
-      set the topLeft of group "iOS Settings" to -1,67
-      set the topLeft of group "iosBuild" to 12,68
-      set the cSize of this card to 1024,676
+      set the topLeft of group "buildbutton" to 50,60
+      set the topLeft of group "iosSettings" to 5,57
+      repeat for each line tLine in the text of button "iOSTabMenu"
+         local tName2
+         put tLine into tName2
+         if tName is "Requirements" then
```

Comment:
This should be `tName2`. This fixes the bug described in the following scenario:

 Recipe:
1. Reset Prefs to default
2. Open a stack and go to iOS settings. The Commercial Tip is shown, as expected. 
3. Close the tip. The "Basic Settings" pane is shifted up, as expected. 
4. Switch to "Requirements" pane.
Expected result: To be shifted up.
Observed result:  It is not shifted up.